src/font/lv_font_fmt_txt.c

```diff
@@ -22,6 +22,11 @@
     #define font_rle LV_GLOBAL_DEFAULT()->font_fmt_rle
 #endif /*LV_USE_FONT_COMPRESSED*/
 
+#if LV_FONT_FMT_TXT_CACHE_GLYPH_CNT > 0
+    #define font_bitmap_cache LV_GLOBAL_DEFAULT()->font_bitmap_cache
```

Comment:
Are those meant to be read_only? if so why not:

```suggestion
    #define font_bitmap_cache() LV_GLOBAL_DEFAULT()->font_bitmap_cache
```
To imply this is a macro ?

Reply:
This is not read-only, we need to create a cache manager object and store it on this pointer.

Reply:
Nice, fair enough.